src/CoreWCF.Kafka/tests/Resources/rabbitmqTestMessage.bin

```diff
@@ -0,0 +1 @@
+<s:Envelope xmlns:s="http://www.w3.org/2003/05/soap-envelope" xmlns:a="http://www.w3.org/2005/08/addressing"><s:Header><a:Action s:mustUnderstand="1">http://tempuri.org/ITestContract/Create</a:Action><VsDebuggerCausalityData xmlns="http://schemas.microsoft.com/vstudio/diagnostics/servicemodelsink">uIDPo4IM/u5L8tFGizKmemV7kmEAAAAAcStFY6F8ukKPmzryw1DTrsJZH/CtAKlEjIDYla6EnwYACQAA</VsDebuggerCausalityData></s:Header><s:Body><Create xmlns="http://tempuri.org/"><name>TestMessage</name></Create></s:Body></s:Envelope>
```

Comment:
Insert a guid to assert.
Remove VsDebuggerCausalityData.
Extract to template string.